### ISSUE TITLE
Error while building APK - Fragments Constructors

### ISSUE THREAD

**Issue by ivanviragine** [AppLozic/Applozic-Android-SDK#236]
I'm getting this error when building signed APK for production (latest version of SDK):

Error:(48) Error: Avoid non-default constructors in fragments: use a default constructor plus Fragment#setArguments(Bundle) instead [ValidFragment]

This should be fixed as denoted here: http://stackoverflow.com/questions/12062946/why-do-i-want-to-avoid-non-default-constructors-in-fragments

**Comment by ivanviragine** [comment]
I see that this has been assigned, but I wonder if is there any due date for the fix, I was planning on publishing today, but because of this error, which only happens when signing APK, I can't. So I need to know when you think you'll be able to fix this.

**Comment by Sunilkumarr** [comment]
@ivanviragine For now you can add this in your app build.gradle file  i will updated that fragment 
```
 android {
          lintOptions {
              checkReleaseBuilds false
          }
      }
```

**Comment by ivanviragine** [comment]
@Sunilkumarr I'm aware of this option, but, those checks are there because using parameters from constructors doesn't guarantee that values will be there in case fragment/activity goes background and/or gets evicted... this lands to a lot of errors for the user...

**Comment by Sunilkumarr** [comment]
Closing this  is fixed in v4.84 https://github.com/AppLozic/Applozic-Android-SDK/commit/c46c455946977ab78697bb8db4cfa69bfa4ff113
